### ISSUE TITLE
Feature Request - Simplify Move Container

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7464]
**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

From object tracking meeting yesterday

https://arctos.database.museum/moveContainer.cfm needs to be simplified and the actions moved to bulk edit tools

![image](https://github.com/ArctosDB/arctos/assets/5725767/034bbec8-e6c6-47f4-850f-2b809556f7e4)

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

This tool is too complicated, allowing those with manage_container under the (hopefully) new container roles #7127  to make pretty significant messes. We believe it should be simplified. Many of the actions here can be accomplished with container bulk edit - https://arctos.database.museum/loaders/bulkEditContainer.cfm

**Describe the solution you'd like**
_How might we accomplish your goals?_

We propose removing everything from this tool except the move function.

![image](https://github.com/ArctosDB/arctos/assets/5725767/0705d121-4179-4e70-a1ff-ff88864812c8)

The only action currently on this form that cannot be completed with container bulk edit is changing part disposition

![image](https://github.com/ArctosDB/arctos/assets/5725767/909077b6-f589-4ee4-9118-a484770a8f46)

which can be accomplished using the part bulk edit tool - https://arctos.database.museum/loaders/bulk_edit_parts.cfm

However, that tool does not accept part barcode, and it would be a lot easier to use in conjunction with container bulk edit if it did. Another issue will be filed for that request, which should be implemented in conjunction with this.

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

Do nothing and wait until this form is misused.

**Additional context**
_Add any other context or screenshots about the feature request here._

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by dustymc** [comment]
>too complicated

To use AND to maintain. I think we can/must elevate this to "necessary."

Can be next task, but blocked by https://github.com/ArctosDB/arctos/issues/7127

**Comment by dustymc** [comment]
issues meeting - this is OK with everyone, go

**Comment by DerekSikes** [comment]
We at UAM entomology used this csv bulk container move tool a LOT. It was simple, scan parent, scan children, click button.

Now you're asking me to train lab techs who have never bulkloaded to start bulkloading to do this same simple task. This is much more complicated - download the template, figure out how to get barcodes into a column (easy to scan them horizontally, but not vertically - this involves learning another tool, a text editor and how to first scan barcodes, then replace either the tabs or the commas (depending on how the scanner is programed) to returns, then copy back to template, then save, then upload (and if no status column in template the bulkoad file will need more manipulations and trainings and things can get stuck or forgotten about).

This has removed a simple 3-step tool and replaced it with a complicated 6+ step process with many more opportunities for user mistakes to be made (and far greater training challenges, esp with different OS systems), slowing down and complicating our workflow.

I get that back-end maintenance is a concern, but why is that given higher priority than ease of use by the actual users?

In another issue I had made regarding this tool I had pointed out how it could be improved to make it even easier to use. Instead, it's gone and made much more complicated.

**Comment by DerekSikes** [comment]
I'll add that currently we were using the bulk container move tool 1-3 times per hour. Bulkloading seems best for less frequent tasks.

**Comment by dustymc** [comment]
>scan parent, scan children, click button

That's what [Container: Move](https://arctos.database.museum/moveContainer.cfm) does (except the clicking is optional, and there's real-time feedback for any problems).

If that doesn't work for some reason, please open a new issue - things will get lost down here.

**Comment by DerekSikes** [comment]
Follow up on this new tool. I used it last Friday to object track 3069 containers (mostly vials). This took 78 minutes. I am ok with changing my scanner from tab to comma and back again. The only problem I noted is that on two occassions my scans were faster than the response times of the tool and oddly my child-scan would appear in the parent field and create errors (which I quickly saw due to the red highlighting). I learned to slow down my scans to a speed that the web could keep up with (but worry that this speed could vary depending on internet traffic and other variables). I still think the prior design, which allows one to scan a csv list directly into a big field & then click one button to save is better (and miss it).

[2024-03-29T23:49:54.330Z]: ERROR: query returned no rows Where: PL/pgSQL function movecontainerbybarcode(character varying,character varying,character varying,character varying) line 69 at SQL statement
[2024-03-29T23:49:52.739Z]: ERROR: query returned no rows Where: PL/pgSQL function movecontainerbybarcode(character varying,character varying,character varying,character varying) line 70 at SQL statement

**Comment by DerekSikes** [comment]
Update from my lab tech Mary: "I have found that the Container: Move tool in Arctos works fine to scan multiple specimens into a unit tray most of the time.  However, it doesn't actually move specimen barcodes to the unit tray if the records are still in the bulkloader, even though it looks like it they are moved. You also don't get the nice output that let you know if a barcode was associated with duplicate records."

Can these two issues be fixed to return the prior functionality?

1) the ability to move barcodes for records that are still in the bulkloader
2) feedback on if the barcode is associated with multiple records

**Comment by dustymc** [comment]
re: https://github.com/ArctosDB/arctos/issues/7464#issuecomment-2030808803 - complain to your network folks, then file an issue if that doesn't work. I just tested a bunch of stuff (in test, which is on average maybe 10 times slower than prod) and I can't make ANYTHING take longer than about 40 milliseconds; I don't think that will be anything I can tune out. 

re: https://github.com/ArctosDB/arctos/issues/7464#issuecomment-2040583246

1. That does not make sense to me, the tool moves containers with barcodes, parts cannot have barcodes.
2. I don't know what this means.

re: both: see https://github.com/ArctosDB/arctos/issues/7464#issuecomment-2015161441, things are still going to get lost down here.

**Comment by DerekSikes** [comment]
This is the correct github for these comments because we are talking about the  Container: Move tool.

I'm checking on item 1 in the prior post - I tested this and it seems to work to me (one can move a barcode even the specimen hasn't loaded from the bulkloader).

Re: #2 - the old bulk move tool, after clicking the done button, gave a list of barcodes moved and the GUIDs associated with those barcodes. This feedback was very helpful in finding mistakes (when one barcode is accidentally assigned to two GUIDs) or barcodes that have no GUIDs associated etc. Can that functionality be returned?
